src/cms/src/components/contact/link.json

```diff
@@ -0,0 +1,29 @@
+{
+  "collectionName": "components_contact_links",
+  "info": {
+    "displayName": "Link",
+    "icon": "link",
+    "description": ""
+  },
+  "options": {},
+  "attributes": {
+    "icon": {
+      "type": "enumeration",
+      "enum": [
+        "phone",
+        "envelope",
+        "laptop",
+        "messages"
+      ],
+      "required": true
+    },
+    "linkText": {
+      "type": "string",
+      "required": true
+    },
+    "linkUrl": {
```

Comment:
We should make a note to add help text for this field after deployment.

e.g. 
Email and phone URLs should NOT include as scheme (e.g. mailto/tel). If you leave this field blank, the linkText will be rendered as plain text with no link.